Avalonia.MusicStore/Views/MainWindow.axaml

```diff
@@ -26,23 +26,26 @@
     </ExperimentalAcrylicBorder>
     <Panel Margin="40">
       <Button HorizontalAlignment="Right" VerticalAlignment="Top"
-        Command="{Binding BuyMusicCommand}">
+        Command="{Binding AddAlbumCommand}">
             <PathIcon Data="{StaticResource store_microsoft_regular}" />
       </Button>
     
-    <ItemsControl Margin="0 40 0 0" ItemsSource="{Binding Albums}">
-      <ItemsControl.ItemsPanel>
-        <ItemsPanelTemplate>
-          <WrapPanel />
-        </ItemsPanelTemplate>
-      </ItemsControl.ItemsPanel>
+      <ScrollViewer VerticalScrollBarVisibility="Auto" Margin="0 40 0 0">
+        <ItemsControl ItemsSource="{Binding Albums}">
+          <ItemsControl.ItemsPanel>
+            <ItemsPanelTemplate>
+              <WrapPanel />
+            </ItemsPanelTemplate>
+          </ItemsControl.ItemsPanel>
+
+          <ItemsControl.ItemTemplate>
+            <DataTemplate>
+              <views:AlbumView Margin="0 0 20 20" />
+            </DataTemplate>
+          </ItemsControl.ItemTemplate>
+        </ItemsControl>
+      </ScrollViewer>
 
-      <ItemsControl.ItemTemplate>
-        <DataTemplate>
-          <views:AlbumView Margin="0 0 20 20" />
-        </DataTemplate>
-      </ItemsControl.ItemTemplate>
-    </ItemsControl>
   </Panel>
```

Comment:
We should fix the formatting in this document I think. Should probably add e.g. this [`.editorconfig`](https://github.com/AvaloniaUI/Avalonia/blob/master/.editorconfig) and use the IDE to reformat the XAML files.